src/vector/index.ts

```diff
@@ -1,21 +1,12 @@
 /*
-Copyright 2015, 2016 OpenMarket Ltd
-Copyright 2017 Vector Creations Ltd
-Copyright 2018, 2019 New Vector Ltd
-Copyright 2019 Michael Telatynski <7t3chguy@gmail.com>
 Copyright 2020 The Matrix.org Foundation C.I.C.
+Copyright 2019 Michael Telatynski <7t3chguy@gmail.com>
+Copyright 2018-2024 New Vector Ltd.
+Copyright 2017 Vector Creations Ltd
+Copyright 2015, 2016 OpenMarket Ltd
```

Comment:
This one doesn't look right?

Reply:
yeah, this looks weird. the lines should be ordered by newest date, not oldest date - NVL shouldn't be buried in the depths.

Reply:
i.e. 2018-2024 should sort at higher precedence to 2019, not lower.